api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/CompletePaymentMessageProcessor.java

```diff
@@ -34,10 +44,12 @@ public MessageType supportsMessageType() {
         return COMPLETE_PAYMENT;
     }
 
-    @Override
-    @Transactional(Transactional.TxType.REQUIRES_NEW)
-    public void processFailedMessage(Message message, FailureEvent failureEvent) {
-        // TODO AFHS-405 add failure when implementing
+    private void sendNotificationEmail(CompletePaymentMessageContext messageContext) {
+        if (messageContext.getPaymentType() == PaymentType.RESTARTED_PAYMENT) {
```

Comment:
I'm just wondering if this needs to be a switch statement because we should probably have a default if its not either of these PaymentTypes, or do we not care if its not?

Reply:
For other types (interim and first payment) we don't send any emails.